src/uart/emsuart_esp32.h

```diff
@@ -26,7 +26,11 @@
 
 #define EMS_MAXBUFFERSIZE 33 // max size of the buffer. EMS packets are max 32 bytes, plus extra for BRK
 
-#define EMSUART_NUM UART_NUM_2 // on the ESP32 we're using UART2
+#ifdef ARDUINO_LOLIN_C3_MINI
+    #define EMSUART_NUM UART_NUM_1 // on C3 mini we're using UART1
```

Comment:
I think you have already tested the code. Don't you have conflicts with the serial console on uart1?
Disabling the serial console is [in emsesp.cpp](https://github.com/emsesp/EMS-ESP32/blob/70efa4f4e65331bba8dc9fa88d2bc6707bf9b7eb/src/emsesp.cpp#L1282).

Reply:
Isn't the serial console on uart0 by default? c3 has 2 ports, uart0 and uart1, where uart0 is for uploading firmware and serial log: https://docs.espressif.com/projects/esp-at/en/latest/esp32c3/Get_Started/Hardware_connection.html

Reply:
You're right, serial is uart0, sorry for confusing.